Modules/Devices/src/Devices.Infrastructure/Persistence/Database/DevicesDbContext.cs

```diff
@@ -63,13 +63,14 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     public async Task RunInTransaction(Func<Task> action, List<int> errorNumbersToRetry,
         IsolationLevel isolationLevel = IsolationLevel.ReadCommitted)
     {
-        ExecutionStrategy? executionStrategy;
+        ExecutionStrategy executionStrategy;
         switch (Database.ProviderName)
         {
             case SQLSERVER:
                 executionStrategy = new SqlServerRetryingExecutionStrategy(this, MAX_RETRY_COUNT, MAX_RETRY_DELAY, errorNumbersToRetry);
                 break;
             case POSTGRES:
+                // ReSharper disable once ConditionIsAlwaysTrueOrFalse
```

Comment:
Why? You are passing `null!` right in the next method. So this means that `errorNumbersToRetry` CAN actually be null.